lib/util/git/index.spec.ts

```diff
@@ -36,6 +36,20 @@ describe('platform/git', () => {
     await repo.add(['future_file']);
     await repo.commit('future message');
 
+    await repo.checkoutBranch('renovate/modified_branch', 'master');
```

Comment:
This is broken and causes the test failures, see https://github.com/steveukx/git-js/issues/490.

We need 2.14.0 to fix that.

Reply:
https://github.com/steveukx/git-js/blob/master/CHANGELOG.md#2140---bug-fix-gitcheckoutbranch-fails-to-pass-commands-to-git-child-process

Reply:
#6818